src/app/backup/destination/destination.config-utilities.ts

```diff
@@ -48,6 +48,7 @@ export type DestinationConfigEntry = {
   description: string;
   customKey?: string;
   oauthField?: string;
+  oauthV2Field?: string;
```

Comment:
This fires off my spider tingles, would we need two differnt oauthFields for the same input it feels it should live elsewhere?